WalletWasabi/Wallets/LockTimeSelector.cs

```diff
@@ -0,0 +1,50 @@
+using NBitcoin;
+using System;
+
+namespace WalletWasabi.Wallets
+{
+	/// <summary>
+	/// Produces random <see cref="LockTime"/> value for a new transaction based on observed lock-time values distribution in Bitcoin mainnet network.
+	/// </summary>
+	/// <remarks>Helps avoid fingerprinting of Wasabi Wallet transactions.</remarks>
+	public class LockTimeSelector
+	{
+		public static LockTimeSelector Instance { get; }
```

Comment:
Properties should be placed under constructors.

Reply:
Fixed.